editoast/src/views/timetable/stdcm.rs

```diff
@@ -192,12 +205,13 @@ async fn stdcm(
     let latest_simulation_end = stdcm_request.get_latest_simulation_end(simulation_run_time);
 
     // 3. Get scheduled train requirements
-    let simulations: Vec<_> = train_simulation_batch(
+    let simulations: Vec<_> = consist_train_simulation_batch(
```

Comment:
Using `consist_train_simulation_batch` doesn't work. We need to use the rolling stocks of timetable train schedules here, so `train_simulation_batch`

> [!CAUTION]
> Doing this should break our CI since STDCM is not working properly. The code as it is, won't be able to compute pathfinding for all train schedules that don't use the rolling stock provided by the STDCM query.
> So the STDCM computation will ignore almost all train schedules.